xml/tuning-ceph-tuning.xml

```diff
@@ -0,0 +1,721 @@
+     before taking effect. It is also possible to deploy these files before
+     running stage 2 of the &productname; deployment process. It
+     is especially desirable to do so if changing the settings that
+     require node or device re-deployment.
+   </para>
+ </sect1>
+ <sect1 xml:id="tuning-core">
+   <title>Core</title>
+   <sect2>
+     <title>Logging</title>
+     <para>
+       It is possible to disable all logging to reduce latency
+       in the various codepaths.
+     </para>
+     <warning>
+       <para>
+         This tuning should be used with caution and understanding that
+         logging <emphasis>will</emphasis> need to be re-enabled should support be required.
+         This implies that an incident would need to be reproduced
+         <emphasis>after</emphasis> logging is re-enabled.
+       </para>
+     </warning>
+<screen>
+  debug ms=0
+  debug mds=0
+  debug osd=0
+  debug optracker=0
+  debug auth=0
+  debug asok=0
+  debug bluestore=0
+  debug bluefs=0
+  debug bdev=0
+  debug kstore=0
+  debug rocksdb=0
+  debug eventtrace=0
+  debug default=0
+  debug rados=0
+  debug client=0
+  debug perfcounter=0
+  debug finisher=0
+</screen>
+   </sect2>
+   <sect2>
+     <title>Authentication parameters</title>
+     <para>
+       Under certain conditions where the cluster is physically secure
+       and isolated inside a secured network with no external exposure,
+       it is possible to disable &cephx;.
+       There are two levels at which &cephx; can be disabled. The first
+       is to disable signing of authentication traffic. This can be
+       accomplished with the following settings:
+     </para>
+<screen>
+cephx_require_signatures = false  (needs test)
+cephx_cluster_require_signatures = false  (needs test)
+cephx_sign_messages = false  (needs test)
+</screen>
+     <para>
+       The second level of tuning completely disables &cephx; authentication.
+       This should only be done on networks that are isolated from public
+       network infrastructure.
+       This change is achieved by adding the following three lines in the
+       global section:
+     </para>
+<screen>
+auth cluster required = none
+auth service required = none
+auth client required = none
+</screen>
+   </sect2>
+   <sect2>
+     <title>RADOS Operations</title>
+     <para>
+       The backend processes for performing RADOS operations show up
+       in <literal>throttle-*objector_ops</literal> when dumping various daemons. If
+       there is too much time being spent in wait, there may be some
+       performance to gain by increasing the memory for in-flight
+       ops or by increasing the total number of inflight operations overall.
+     </para>
+<screen>
+objecter inflight op bytes = 1073741824 # default 100_M
+objecter inflight ops = 24576
+</screen>
+   </sect2>
+   <sect2>
+     <title>OSD parameters</title>
+     <para>
+       Increasing the number of op threads may be helpful with SSD
```

Comment:
Maybe wrap `op threads` in [literal] or highlight it in some other way? It reads oddly to me without.

Reply:
whatever works.